crd/fleet/fleet-crds.yaml

```diff
@@ -1813,26 +1812,26 @@ spec:
                             type: array
                           requiredDuringSchedulingIgnoredDuringExecution:
                             description: If the affinity requirements specified by
-                              this field are not met at scheduling time, the pod will
+                              this field isn't met at scheduling time, the pod will
                               not be scheduled onto the node. If the affinity requirements
-                              specified by this field cease to be met at some point
-                              during pod execution (e.g. due to a pod label update),
-                              the system may or may not try to eventually evict the
+                              specified by this field stop to be met at some point
+                              during pod execution - example, due to a pod label update -
```

Comment:
Add "for" before "example" -- so, it will be "during pod execution -- for example, due to a pod label update -"